projects/packages/my-jetpack/_inc/components/product-cards-section/protect-card/index.tsx

```diff
@@ -21,8 +21,8 @@ const ProtectCard: FC< { admin: boolean } > = ( { admin } ) => {
 	}, [ recordEvent ] );
 
 	const shouldShowSecondaryButton = useCallback(
-		() => ! hasProtectPaidPlan,
-		[ hasProtectPaidPlan ]
+		() => isPluginActive && ! hasProtectPaidPlan,
+		[ hasProtectPaidPlan, isPluginActive ]
```

Comment:
It seems to me we're using the multiple button pattern (2 buttons, 1 primary & 1 secondary) only when a plugin/product is "owned" but also it has the ability to be upgraded to a higher or paid plan.  With that case we show an "Upgrade" button (to take advantage of the opportunity to sell or upsell the plan), but we also still show a "View" button so that the user can still navigate to the product page and interact with the particular tier (free or paid) that they currently have.
It seems to only be for this situation, so far, that we are using a side by side primary & secondary button. Or at least that is how I understand it, AFAIK.
But at the same time, I don't see why we couldn't or shouldn't use multiple buttons in other in other ways.. I suppose it's just a design preference and/or functionality preference decision. 🤷‍♂️

Reply:
Interesting 🤔 @ilonagl was wondering if you had any insight on this? The Backup card is using a dropdown for its multiple functionality, should we maybe start using multiple buttons there, and anywhere else we want multiple CTAs? Or is the pattern that we should use multiple buttons in this way only and a dropdown when the CTAs are more custom?